website_form_recaptcha/static/src/js/field_recaptcha.js

```diff
@@ -6,7 +6,6 @@ odoo.define('website_form_recaptcha.recaptcha', function(require){
   "use strict";
```

Comment:
I know you didn't really create these warnings, but while here it might be nice to fix:
```
website_form_recaptcha/static/src/js/field_recaptcha.js:7: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]
website_form_recaptcha/static/src/js/field_recaptcha.js:9: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]
website_form_recaptcha/static/src/js/field_recaptcha.js:11: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]
website_form_recaptcha/static/src/js/field_recaptcha.js:20: [W7903(javascript-lint), ] Assignment to function parameter 'data'. [Error/no-param-reassign]
website_form_recaptcha/static/src/js/field_recaptcha.js:30: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]
website_form_recaptcha/static/src/js/field_recaptcha.js:32: [W7903(javascript-lint), ] Trailing spaces not allowed. [Error/no-trailing-spaces]
```